UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/ReactionManager.cs

```diff
@@ -58,7 +59,8 @@ private void Awake()
 			matrix = GetComponent<Matrix>();
 
 			hotspots = new ConcurrentDictionary<Vector3Int, MetaDataNode>();
-			winds = new UniqueQueue<MetaDataNode>();
+			windsTwo =  new ThreadSafeList<MetaDataNode>();
+			windsNodeDelegator = ProcessWindNodes;
```

Comment:
why this assignation and not just use the function directly?

Reply:
saves some extra gc but yes its possible to send the function directly

Reply:
the assignation saves gc? I need to learn how this gc shit works.